src/app/models/tech-record.model.ts

```diff
@@ -86,14 +94,46 @@ interface AxleBrakes {
   springBrakeParking: boolean;
 }
 
-interface Axle {
+export interface Axle {
   axleNumber: number;
   parkingBrakeMrk: boolean;
   weights: Weights;
   tyres: Tyres;
   brakes: AxleBrakes;
 }
 
+export interface PurchaserDetails {
+  name: string;
+  address1: string;
+  address2: string;
+  postTown: string;
+  address3: string;
+  postCode: string;
+  emailAddress: string;
+  telephoneNumber: string;
+  faxNumber: string;
```

Comment:
we should refactor this to addressDetails and reuse for ManufactureDetails

Reply:
refactoerd